calculator.app/calculator.py

```diff
@@ -0,0 +1,18 @@
+def add(a, b):
```

Comment:
- Use parameter name that is more meaningful. Example `value_1` , `value_2`. 
- Please add type hint for every parameter too.